slides.md

```diff
@@ -156,57 +156,99 @@ import anotherFunction, {aFunction} from "./someFile";
 
 ---
 # Hello Todo!
-1. Edit `App.js` so that the list rendered from an array:
+Currently, `App.js` is quite boring, it would be great if instead of having the todos hardcoded, we could somehow cycle through an array of todos and show a `Text` per element.
+
+First of all, we need to extract the data for the todos, in `App.js`:
 ```javascript
 const todos = [{
     name: "Take the dog out",
-  }, ...
+  }, // ...
 ];
 ```
-You can use `todos.map()` to return a component, return a `Text` component showing `todo.name`.
+
+Inside `render()`, you can then map those todos to a certain element:
```

Comment:
If this is a "for dummies" maybe you want to explain why `render()` and what it is, or at least add a link explaining it

Reply:
ec385ab